### PR TITLE
StringUtil Abbreviation method fixed

### PR DESCRIPTION
**Pull request by isaacg97**
Abbreviation method was returning the string in full if abbrevMarker was ""

was this
`if (isEmpty(str) || isEmpty(abbrevMarker ) {
            return str;
        }`

needed to be this
`if (isEmpty(str) || abbrevMarker == null ) {
            return str;
        }`

### CODE REVIEW

File: src/test/resources/java.policy [hunk 1]
```diff
@@ -1,375 +0,0 @@
-//
```
Comment (Stzx) [inline src/test/resources/java.policy]: Why delete the file?

File: src/test/resources/lang-708-input.txt [hunk 2]
```diff
@@ -1 +0,0 @@
...ija","nah":"Andalucia","lad":"Andaluziya","de":"Andalusien","als":"Andalusien","qu":"Andalusiya","hy":"Անդալուզիա","eu":"Andaluzia","is":"Andalúsía","uk":"Андалусія","az":"Andalusiya","mr":"आंदालुसिया","ug":"Andalusiye","fa":"اندلس","ar":"أندلوسيا","rmy":"Andalusiya","io":"Andaluzia","el":"Ανδαλουσία","frp":"Andalosie","pt":"Andaluzia","eo":"Andaluzio","en":"Andalusia","et":"Andaluusia","es":"Andalucía","ja":"アンダルシア州","nl":"Andalusië","af":"Andalusië","vec":"Andalusìa"},"geonameFeatureCode":"ADM1","_id":261,"name":"","auto":true,"type":"GEO","geonameId":2593109,"valueCode":""},{"geonameFeatureClass":"A","values":{"de":"Granada","default":"Province of Granada","fr":"Grenade","en":"Province of Granada","es":"Provincia de Granada","ja":"グラナダ"},"geonameFeatureCode":"ADM2","_id":262,"name":"","auto":true,"type":"GEO","geonameId":2517115,"valueCode":""},{"geonameFeatureClass":"A","values":{"default":"Monachil"},"geonameFeatureCode":"ADM3","_id":263,"name":"","auto":true,"type":"GEO","geonameId":6357744,"valueCode":""},{"geonameFeatureClass":"P","values":{"default":"Sierra Nevada"},"geonameFeatureCode":"PPL","_id":264,"name":"","auto":true,"type":"GEO","geonameId":6544329,"valueCode":""}]
```
Comment (Stzx) [inline src/test/resources/lang-708-input.txt]: Why delete the file?

File: src/test/java/org/apache/commons/lang3/StringUtilsTest.java [hunk 3]
```diff
@@ -323,6 +323,7 @@ public void testAbbreviate_StringStringInt() {
                 "StringUtils.abbreviate expecting IllegalArgumentException");
     }
 
+
```
Comment (Stzx) [inline src/test/java/org/apache/commons/lang3/StringUtilsTest.java]: Extra line.

File: src/main/java/org/apache/commons/lang3/StringUtils.java [hunk 4]
```diff
@@ -362,6 +363,8 @@ public static String abbreviate(final String str, final String abbrevMarker, int
             return abbrevMarker + abbreviate(str.substring(offset), abbrevMarker, maxWidth - abbrevMarkerLength);
         }
         return abbrevMarker + str.substring(str.length() - (maxWidth - abbrevMarkerLength));
+
```
Comment (Stzx) [inline src/main/java/org/apache/commons/lang3/StringUtils.java]: Extra line.

**Comment by garydgregory** [comment]
Closing: No reply in years.

**Pull request closed without merging**